components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -197,9 +197,26 @@ - (CGSize)sizeForChildContentContainer:(id<UIContentContainer>)container
     return CGSizeZero;
   }
 
+  // For pre iOS 11 devices, we are assuming a safeAreaInset of UIEdgeInsetsZero
```

Comment:
Do we need to react to viewSafeAreaInsetsDidChange?

Reply:
This initially appears tricky.  
View controllers have viewSafeAreaInsetsDidChange, but presentation controllers do not.  I'll explore further.